lib/batch/batch_metadata_updates_634.rb

```diff
@@ -0,0 +1,73 @@
+# batch updates for https://github.com/galterlibrary/digital-repository/issues/634
+
+# Updates number 1:
+#   - All items with "Pediatric Neurology Briefs Publishers" in the Original 
+#   - Publisher field --> Add additional field to Digital Publisher with "Pediatric 
+#   - Neurology Briefs Publishers" and delete from Original Publisher
+
+pnb_publishers = "Pediatric Neurology Briefs Publishers"
+items_needing_original_publisher_updates = GenericFile.where("original_publisher" => pnb_publishers) + 
+                                           Collection.where("original_publisher" => pnb_publishers)
+
+items_needing_original_publisher_updates.each do |item|
+  item.publisher += [pnb_publishers]
```

Comment:
Should we check that publisher contains `png_publishers` already? Is that a possibility?

Reply:
I checked and it won't save duplicate, so I think we're safe checking if it exists already.

Reply:
Interesting, `item.publisher` is a `Set` or is filtered out of redundancies on save? How did you check?

Reply:
I checked this by manually testing it. I will look to see if there is a test that checks this scenario, and add one if not.